.github/workflows/daily-e2e-tests.yml

```diff
@@ -21,38 +21,44 @@ jobs:
           - 9.3.16
           - 9.4.13
           - 9.5.7
-          - 10.0.2
-
-          # NOTE: don't run on > 10.0.3 until this issue is fixed. It appears that something was changed
-          # with the plugin-proxy that caused it to start throwing HTTP 502s from time to time
-          # https://raintank-corp.slack.com/archives/C01C4K8DETW/p1692279329797149
+          - 10.0.11
+          - 10.1.7
+          # TODO: fix issues with running e2e tests against Grafana v10.2.x and v10.3.x
+          # - 10.2.4
           # - latest
       fail-fast: false
     uses: ./.github/workflows/e2e-tests.yml
     with:
       grafana-image-tag: ${{ matrix.grafana-image-tag }}
       run-expensive-tests: true
-      browsers: "chromium firefox webkit"
+      # TODO: fix issues with some tests on firefox and webkit
+      # example CI build with firefox/webkit failing tests
+      # https://github.com/grafana/oncall/actions/runs/8022194346/job/21915964672#step:19:905
+      # browsers: "chromium firefox webkit"
```

Comment:
Can we create a task and put it in proposal for next sprint to not delay this one?

Reply:
great idea! https://github.com/grafana/oncall/issues/3962